R/oauth-cache.R

```diff
@@ -0,0 +1,245 @@
+## this file has its origins in oauth-cache.R in httr
+## nothing there is exported, so copied over, then it evolved
+
+# cache --------------------------------------------------------------
+
+gargle_default_oauth_cache_path <- "~/.R/gargle/gargle-oauth"
```

Comment:
Should we be using keyring for this?

Reply:
Yes, probably, I just couldn't bear to think about that yet.